website/content/v1.8/introduction/getting-started.md

```diff
@@ -182,9 +174,23 @@ to reflect `vda` instead of `sda`.
 
 > For information on customizing your machine configurations (such as to specify the version of Kubernetes), using [machine configuration patches]({{< relref "../talos-guides/configuration/patching" >}}), or customizing configurations for individual machines (such as setting static IP addresses), see the [Production Notes]({{< relref "prodnotes#customizing-machine-configuration" >}}).
 
-## Understand talosctl, endpoints and nodes
+## Accessing the Talos API
+
+Administrative tasks are performed by calling the Talos API (usually with `talosctl`) on Talos Linux control plane nodes, who may forward the requests to other nodes.
+Thus:
+- ensure your control plane node is directly reachable on TCP port 50000 from the workstation where you run the `talosctl` client.
+- Until a node is a member of the cluster, it does not have the PKI infrastructure set up, and so will not accept API requests that are proxied through a control plane node.
```

Comment:
it's actually the other way around - controlplane nodes would refuse to talk to the nodes which don't have right PKI (they can't establish mutual TLS while trying to forward the API request)